Components/Mirrored.js

```diff
@@ -1,10 +1,11 @@
 import React from 'react';
 
-const Mirrored = ({ children, id, hover, hide }) => {
+const Mirrored = ({ children, id, hover, hide, disable }) => {
   const finalProps = {
     'data-mirror-id': id,
     'data-mirror-hide': hide,
     'data-mirror-hover': hover,
+    'data-mirror-disable': disable,
```

Comment:
disable uses `display: none`. visiblity hidden was not working here, probably because we're rendering an iframe